modules/enabling-gpu-support-in-data-science.adoc

```diff
@@ -53,15 +64,25 @@ The *Delete ConfigMap* dialog appears.
 .. Inspect the dialog and confirm that you are deleting the correct ConfigMap.
 .. Click *Delete*.
 . Restart the dashboard replicaset.
+ifdef::upstream,self-managed[]
 .. In the {openshift-platform} web console, change into the *Administrator* perspective.
+endif::[]
+ifdef::cloud-service[]
+.. In the OpenShift web console, change into the *Administrator* perspective.
+endif::[]
 .. Click *Workloads* -> *Deployments*.
 .. Set the *Project* to *All Projects* or *redhat-ods-applications* to ensure you can see the appropriate deployment.
 .. Search for the *rhods-dashboard* deployment.
 .. Click the action menu (&#8942;)  and select *Restart Rollout* from the list.
 .. Wait until the *Status* column indicates that all pods in the rollout have fully restarted.
 
 .Verification
+ifdef::upstream,self-managed[]
 * The NVIDIA GPU Operator displays on the *Operators* -> *Installed Operators* page in the {openshift-platform} web console.
```

Comment:
Replace "displays" with "appears". _Displays_ is a transitive verb in this context, so it is wrong to use it without an accompanying object.